jujugui/static/gui/src/app/components/import-export/test-import-export.js

```diff
@@ -167,4 +178,48 @@ describe('ImportExport', function() {
     assert.equal(importBundleFile.callCount, 1);
     assert.equal(importBundleFile.args[0][0], 'apache2.yaml');
   });
+
+  it('can disable importing when read only', function() {
+    acl.isReadOnly = sinon.stub().returns(true);
+    var currentChangeSet = {one: 1, two: 2};
+    var renderer = jsTestUtils.shallowRender(
+      <juju.components.ImportExport
+        acl={acl}
+        changeState={sinon.stub()}
+        currentChangeSet={currentChangeSet}
+        exportEnvironmentFile={sinon.stub()}
+        hasEntities={true}
+        hideDragOverNotification={sinon.stub()}
+        importBundleFile={sinon.stub()}
+        renderDragOverNotification={sinon.stub()} />, true);
+    var instance = renderer.getMountedInstance();
+    var output = renderer.getRenderOutput();
+    var expected = (
+      <div className="import-export">
+        <span className="import-export__export link"
+          onClick={instance._handleExport}
```

Comment:
I figured it's probably OK to export in read only mode, so I've only disabled the import button.